app/views/browse/_deployment-details.html

```diff
@@ -16,7 +16,7 @@
         <status-icon status="deployment | deploymentStatus"></status-icon>
         {{deployment | deploymentStatus}}
         <span style="margin-left: 7px;">
-          <button ng-show="!rollBackCollapsed && (deployment | deploymentStatus) == 'Complete' && !(deployment | deploymentIsLatest : deploymentConfig) && !deployment.metadata.deletionTimestamp" ng-disabled="(deploymentConfigDeploymentsInProgress[deploymentConfigName] | hashSize) > 0" type="button" class="btn btn-default btn-xs" ng-click="rollBackCollapsed = !rollBackCollapsed">Roll Back</button>
+          <button ng-if="'deploymentconfigrollbacks' | canI : 'create'" ng-show="!rollBackCollapsed && (deployment | deploymentStatus) == 'Complete' && !(deployment | deploymentIsLatest : deploymentConfig) && !deployment.metadata.deletionTimestamp" ng-disabled="(deploymentConfigDeploymentsInProgress[deploymentConfigName] | hashSize) > 0" type="button" class="btn btn-default btn-xs" ng-click="rollBackCollapsed = !rollBackCollapsed">Roll Back</button>
```

Comment:
Wondering if we want an `ng-if` and an `ng-show` on the same node.  I don't mind the `ng-if` as much, but it seems like the `ng-show` is overloaded already. We should probably have a boolean for this and a clearly named function indicating what all this is doing.

I know that the `ng-show` isn't actually part of your PR, adding the `ng-if` just makes the complexity apparent.